app/views/system_admin/providers/new.html.erb

```diff
@@ -12,6 +12,7 @@
 
       <%= f.govuk_text_field :name, label: { text: "Provider name", size: "s" }, width: 20, autocomplete: :disabled %>
       <%= f.govuk_text_field :dttp_id, label: { text: "DTTP ID", size: "s" }, width: 20, autocomplete: :disabled %>
+      <%= f.govuk_check_box :apply_sync_enabled, true, multiple: false, label: { text: "Import application data from Apply?<br><br>".html_safe, size: "s" } %>
```

Comment:
You can get rid of the line breaks and `html_safe` method call and just wrap the checkbox with `<div class="govuk-form-group"></div>.`

Reply:
Thanks John, I've fixed